examples/autobahn_client.nim

```diff
@@ -80,7 +80,7 @@ proc main() {.async.} =
 
       while ws.readystate != ReadyState.Closed:
         # echo back
-        let data = await ws.recv()
+        let data = await ws.recvMsg()
```

Comment:
This is definitely better than forecast in #86

Reply:
I will upload the rest of whai i did at #89 which is not unlike the this solution here so @dryajov can use fragments or ditch it all together.